inspec.yml

```diff
@@ -0,0 +1,383 @@
+      - "SV-230505" # AC-17 (1)
+      - "SV-244544" # AC-17 (1)
+      - "SV-230223" # AC-17 (2)
+      - "SV-230251" # AC-17 (2)
+      - "SV-230252" # AC-17 (2)
+      - "SV-230254" # AC-17 (2)
+      - "SV-230255" # AC-17 (2)
+      - "SV-230256" # AC-17 (2)
+      - "SV-230257" # AC-17 (2)
+      - "SV-244526" # AC-17 (2)
+      - "SV-230506" # AC-18 (1)
+      - "SV-230507" # AC-18 (1)
+      - "SV-230483" # AU-5 (1)
+      - "SV-244543" # AU-5 (1)
+      - "SV-230475" # AU-9 (3)
+      - "SV-230508" # CM-7 (2)
+      - "SV-230509" # CM-7 (2)
+      - "SV-230510" # CM-7 (2)
+      - "SV-230511" # CM-7 (2)
+      - "SV-230512" # CM-7 (2)
+      - "SV-230513" # CM-7 (2)
+      - "SV-230514" # CM-7 (2)
+      - "SV-230515" # CM-7 (2)
+      - "SV-230516" # CM-7 (2)
+      - "SV-230517" # CM-7 (2)
+      - "SV-230518" # CM-7 (2)
+      - "SV-230519" # CM-7 (2)
+      - "SV-230520" # CM-7 (2)
+      - "SV-230521" # CM-7 (2)
+      - "SV-230522" # CM-7 (2)
+      - "SV-230523" # CM-7 (2)
+      - "SV-230545" # CM-7 (2)
+      - "SV-230546" # CM-7 (2)
+      - "SV-230296" # IA-2 (5)
+      - "SV-230502" # IA-3
+      - "SV-230503" # IA-3
+      - "SV-230524" # IA-3
+      - "SV-244547" # IA-3
+      - "SV-244548" # IA-3
+      - "SV-230229" # IA-5 (2)
+      - "SV-230230" # IA-5 (2)
+      - "SV-230355" # IA-5 (2) 
+      - "SV-230240" # SC-3
+      - "SV-230241" # SC-3
+      - "SV-230277" # SC-3
+      - "SV-230278" # SC-3
+      - "SV-230279" # SC-3
+      - "SV-230242" # SC-4
+      - "SV-230243" # SC-4
+      - "SV-230269" # SC-4
+      - "SV-230270" # SC-4
+      - "SV-230526" # SC-8
+      - "SV-244549" # SC-8
+      - "SV-230244" # SC-10
+      - "SV-244525" # SC-10      
+      - "SV-230224" # SC-28
+      - "SV-244540" # SI-2 (2)
+      - "SV-230282" # SI-6
+      - "SV-230245" # SI-11
+      - "SV-230246" # SI-11
+      - "SV-230247" # SI-11
+      - "SV-230248" # SI-11
+      - "SV-230249" # SI-11
+      - "SV-230250" # SI-11
+      - "SV-230276" # SI-16
+      - "SV-230280" # SI-16
+      - "SV-230364" # IA-5 (1) d
+      - "SV-230365" # IA-5 (1) d
+      - "SV-230366" # IA-5 (1) d
+      - "SV-230367" # IA-5 (1) d
+      - "SV-230362" # IA-5 (1) b
+      - "SV-230375" # IA-5 (1) a
+    <% else %>
+    required: true
+    value: []
+    <% end %>
+
+#SV-230244, SV-244525:
+  - name: client_alive_interval 
+    desc: Interval of network connections associated with SSH 
+    type: Numeric
+    value: 1800
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+#SV-230353    
+  - name: system_inactivity_timeout
+    desc: Maximum system inactivity timeout (time in seconds).
+    type: Numeric
+    value: 1800
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+#SV-230331, SV-230374
+  - name: temp_account_expire_time
+    desc: Temporary (short term) should be set with an expiration date/time (time in hours)
+    type: Hash
+    value: 
+      hours: 24
+      days: 1
+    profile: redhat-enterprise-linux-8-stig-baseline
```

Comment:
For `SV-230331`, H: 30 | M: 60 | L: NA | HVA: 0.5
For `SV-230374`, H: 1 | M: 1 | L: NA | HVA: 0.5

Reply:
Done https://github.com/CMSgov/cms-ars-5.0-red-hat-enterprise-linux-8-stig-overlay/pull/1/commits/96fee1639835fd6201e0d2109da44554c5d1d8bc